docs/book/src/reference/creating-events

```diff
@@ -0,0 +1,109 @@
+# Creating Events
+
+It is often useful to publish *Event* objects from the controller Reconcile function. 
+
+Events allow users to see what is going on with a particular object, and allow automated processes to see and respond to them.
+
+{% panel style="success", title="Getting Events" %}
+Recent Events for an object can be viewed by running `kubectl describe`
+{% endpanel %}
+
+{% method %}
+
+###how to implement the solution to raise an event:
+
+a) You need to pass the recorder when you set up the recorder for the controller when the event will be raised see: https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v4-with-deploy-image/main.go#L103
```

Comment:
We cannot add those links. The content on them can change. Instead of that, we should add a snippet of the code. 
Also, please look at the other docs and try to follow/adopt the same standards.

Reply:
https://github.com/Sajiyah-Salat/kubebuilder/blob/master/docs/book/src/migration/legacy/migration_guide_v2tov3.md is this migration guide you are referring to.

Reply:
Links are not working. I tried to change it with code snippets. but its not working now. I think some new pr have affect that.

Reply:
We should not use the links at all here. We should add examples within code.

Reply:
I know that we dont need to use links. but we need to have code snippets. therefore I want to open the link and get that particular line in the form of code snippets. Therefore opening a link is necessary. wdyt?

Reply:
See the suggestion made so that you can better check how we can do in this case.